projects/fmcadc2/common/fmcadc2_bd.tcl

```diff
@@ -1,11 +1,18 @@
+#
+# Parameter description:
+#   RX_JESD_M : Number of converters per link
+#   RX_JESD_L : Number of lanes per link
+#   RX_JESD_S : Number of samples per frame
+#   RX_JESD_NP : Number of bits per sample
+#
 
 source $ad_hdl_dir/library/jesd204/scripts/jesd204.tcl
 
 # JESD204B interface configuration parameters
-set RX_NUM_OF_LANES 8
-set RX_NUM_OF_CONVERTERS 1
-set RX_SAMPLES_PER_FRAME 4
-set RX_SAMPLE_WIDTH 16
+set RX_NUM_OF_LANES  $ad_project_params(RX_JESD_L)
+set RX_NUM_OF_CONVERTERS  $ad_project_params(RX_JESD_M)
+set RX_SAMPLES_PER_FRAME  $ad_project_params(RX_JESD_S)
+set RX_SAMPLE_WIDTH  $ad_project_params(RX_JESD_NP)
 
 set adc_fifo_name axi_ad9625_fifo
 set adc_data_width 256
```

Comment:
I think adc_data_width should be a function of number of lanes, 32*L, for cases when we don't use F=8

Reply:
Changed adc_data_width to be a function of the number of lanes